### PR TITLE
Add autolayout and tests

### PR DESCRIPTION
**Pull request by vadymmarkov**

### CODE REVIEW

File: Source/TutorialModel.swift [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import UIKit
+import Cartography
 
 @objc public class TutorialModel: NSObject {
 
+  struct Dimensions {
+    static let minimumMarginSpace: CGFloat = 20.0
```
Comment (3lvis) [inline Source/TutorialModel.swift]: nice :)

File: Source/TutorialController.swift [hunk 2]
```diff
@@ -40,20 +37,27 @@ let MinimumMarginLateralSpace: CGFloat = 20.0
...
 
+  @objc public static func setTextColor(color: UIColor) {
+    UITextView.appearance().textColor = color
+  }
+
   // MARK: Device orientation
 
   func didRotate() {
-    layoutSubviews()
+    //layoutSubviews()
```
Comment (3lvis) [inline Source/TutorialController.swift]: If this is commented do we even need this method?

File: Pod/Tests/Helpers/ResourceHelper.swift [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import UIKit
+
+class ResourceHelper {
```
Comment (3lvis) [inline Pod/Tests/Helpers/ResourceHelper.swift]: What's this, do we need it?

**Comment by 3lvis** [comment]
Any reason for naming your tests class `TutorialModelSpec.swift` instead of `TutorialModelTests.swift`?

File: Source/TutorialController.swift (hunk 2 above)
Comment (vadymmarkov) [inline Source/TutorialController.swift]: Method is not used now, kept it just for a case we'll need to do something when orientation is changed. But can remove it.

**Comment by 3lvis** [comment]
@vadymmarkov mind fixing the merge conflicts?

File: Pod/Tests/Helpers/ResourceHelper.swift (hunk 3 above)
Comment (vadymmarkov) [inline Pod/Tests/Helpers/ResourceHelper.swift]: Will remove this.

File: Source/TutorialController.swift (hunk 2 above)
Comment (3lvis) [inline Source/TutorialController.swift]: Better to remove unused :)

**Comment by vadymmarkov** [comment]
@3lvis yes, will merge conflicts.
As for `TutorialModelSpec.swift` I think if class extends from `QuickSpec` it's Spec, no Tests :smile: 
And in most of examples I see this name convention.

**Comment by 3lvis** [comment]
@vadymmarkov Interesting!

**Comment by vadymmarkov** [comment]
@3lvis Merged with master.

**Pull request merged**